src/main/CMakeLists.txt

```diff
@@ -58,11 +58,25 @@ endif()
 #-----------------------------------------------------------------------------#
 # cvc5 binary configuration
 
+# Define the binary extension generated by the link process
+if(WASM STREQUAL "WASM")
+  set(BINARY_EXTENSION ".wasm")
+elseif(WASM STREQUAL "JS")
+  set(BINARY_EXTENSION ".js")
+elseif(WASM STREQUAL "HTML")
+  set(BINARY_EXTENSION ".html")
+else()
+  set(BINARY_EXTENSION "")
+endif()
+string(REGEX REPLACE "-s +" "-s" WASM_FLAGS "${WASM_FLAGS}")
```

Comment:
Why is this necessary?

Reply:
It isn't. I was using it to standardize the flags being passed to the link process. I will remove it, didn't realize that was useless.